dangerzone/windows/wsl.py

```diff
@@ -0,0 +1,117 @@
+import functools
+import logging
+import platform
+import subprocess
+from typing import Callable, Optional, Tuple
+
+from .. import errors
+from ..util import subprocess_run
+from . import shellexec
+
+log = logging.getLogger(__name__)
+
+# In the following commands, we set the encoding to non-BOM UTF16, which is what WSL
+# uses. If not set, stdout will contain invisible null characters which renders the
+# output impossible to read and copy/paste.
+# Note that this is not necessary for commands initiated with ShellExecuteEx, since they
+# don't print to a handle that Python controls, but to a terminal spawned by Windows.
+
+
+def wsl_list() -> str:
```

Comment:
I think these functions should be named `install`, `status`, etc, and used with their namespaces:

```python
from dangerzone.windows import wsl

wsl.install() # etc.
```